### ISSUE TITLE
Still broken on 115 ESR

### ISSUE THREAD

**Issue by LuftRaptor** [dimdenGD/OldTwitter#980]
**Describe the bug**
Endlessly loads with just the twitter logo, reinstalling the plugin, or even resetting firefox does not seem to change results
something about a challenge being unable to load, and no crypto key

**Screenshots**
![image](https://github.com/user-attachments/assets/5b1b423e-005b-4723-9d9d-7c2643257a3e)
![image](https://github.com/user-attachments/assets/940fdf2d-7ee9-44ad-aba1-1a5b3704563e)

**Browser**
Firefox 115 ESR

**Links**
x.com/home

**OldTwitter version**
1.8.9.7

**Comment by UnholyEntropy** [comment]
I'm having the same issues as well. Precisely.

**Comment by aubymori** [comment]
same issue, console:
![image](https://github.com/user-attachments/assets/2ea1d810-aabf-430d-9161-21a364878323)

**Comment by 2lipp2p** [comment]
bump

**Comment by panettondoro** [comment]
issue is with firefox's tracking protection. check the solution in #843

**Comment by 2lipp2p** [comment]
> issue is with firefox's tracking protection. check the solution in #843

not working still, am i doing anything wrong? 
![image](https://github.com/user-attachments/assets/2f695455-66bf-4eaf-a29b-76f47f9b07d7)

**Comment by panettondoro** [comment]
not sure actually! i think the issue also lies somewhere in the challenge resolver as well, as every once in a while oldtwitter won't load and keeps logging in the console something akin to "couldn't fetch challenge". i usually solve it by refreshing the page until it works so maybe try doing that!

**Comment by Chancellor2020** [comment]
I'm still infinitely loading so I can't guarantee this works

**Comment by panettondoro** [comment]
yeah, it's bit of a hack. just for reference, are you using a firefox derivate? if so, it might have some changes that prevent the solution from working.

**Comment by JohnJohnMohn** [comment]
bump

**Comment by UnholyEntropy** [comment]
bump

**Comment by dimdenGD** [comment]
I don't expect it to be ever fixed

**Comment by aubymori** [comment]
> I don't expect it to be ever fixed

why not

**Comment by JohnJohnMohn** [comment]
> > I don't expect it to be ever fixed
> 
> why not
I don't want to say they don't care but i'm guessing by the lack of response there probably won't be any fixes soon, maybe even discontinuation.

**Comment by JohnJohnMohn** [comment]
disregard the deleted comment i thought this was about echelon

**Comment by aubymori** [comment]
please

**Comment by Kakuyamii** [comment]
> I don't expect it to be ever fixed

Is there a particular reason to why? Does ESR just have an incompatibility for it?

**Comment by LuftRaptor** [comment]
Ok this may NOT have anythign to do with 115 ESR specifically!  the issue appeared again on r3dfox after i used resource hacker to change the 1033 string to identify as "internet Explorer"  So if any of you have used resource hacker to rename the process i STRONGLY suggest setting it back to firefox or using r3dfox

**Comment by aubymori** [comment]
> after i used resource hacker to change the 1033 string to identify as "internet Explorer"

1033 isn't a string ID. It's the locale ID for en-US.

**Comment by aubymori** [comment]
It's 103. And I seriously doubt that would cause it.

**Comment by dimdenGD** [comment]
This makes me think that you could try spoofing your user agent to like latest Chrome version and see if it helps, cuz I have no idea why it even happens only on 115 ESR

**Comment by Harrendan** [comment]
I can't believe this is still not fixed. I only started having this issue semi recently, but to come here and find out that some people haven't been able to use the addon since December of last year??? I just hope someone gets on this ASAP. The overlap of 115 ESR users and OldTwitter users is massive.

**Comment by UnholyEntropy** [comment]
> I can't believe this is still not fixed. I only started having this issue semi recently, but to come here and find out that some people haven't been able to use the addon since December of last year??? I just hope someone gets on this ASAP. The overlap of 115 ESR users and OldTwitter users is massive.
------------------------------------------------------------------------------------------------------
I couldn't agree more with you! Still waiting unfortunately. :/

**Comment by zeffdakilla** [comment]
having this issue as well since i just switched to 115esr, tried some of the solutions offered and none worked unfortunately
